mobile/src/pages/Authentication.tsx

```diff
@@ -1,14 +1,22 @@
 import { StatusBar } from "expo-status-bar";
 import Button from "../components/Button";
-import { View } from "react-native";
+import { View } from 'react-native';
 
 export default function Authentication({ navigation }) {
+
+  //# region Styles
+
+  const baseStyle = "justify-center items-center";
```

Comment:
I might name this `center` if bothering to extract it at all. Probably just leaving it duplicated is fine because it isn't too much code.